docs/module/srv6.md

```diff
@@ -1,18 +1,61 @@
 # Segment Routing over IPv6 (srv6) Configuration Module
 
-This configuration module configures SRv6 over IS-IS and (optionally) BGP on Nokia SR OS.
+This configuration module configures SRv6 over IS-IS and (optionally) BGP and L3VPN
 
 Supported SRv6 features:
 
 * End SIDs (nodes) and End-x SIDs (links), all statically allocated
 * IS-IS routing protocol with SRv6 TLVs
 * Dynamic BGP Service SIDs
+* BGP L3VPN v4/v6 over SRv6
 
 The module currently depends on IS-IS module and will trigger a configuration error if the **isis** module is not enabled in the network topology.
 
-## Parameters
+## Platform Support
+The following table describes per-platform support of individual router-level SRv6 features:
 
-* Locator: an IPv6 address range to allocate to SRv6 nodes; each node is assigned a unique /64 prefix from this range
+| Operating system         | IS-IS | OSPFv3 | BGP v4/v6 |
+| ------------------------ |:-----:|:------:|:---------:|
+| FRR                      |   ✅  ||  ❌   |     ✅    |
```

Comment:
Extra vertical bar, results in empty OSPFv3 column and unsupported BGP v4/v6 column